wrapper/src/main/java/software/amazon/jdbc/ConnectionPlugin.java

```diff
@@ -128,7 +128,7 @@ Connection forceConnect(
    * @throws UnsupportedOperationException if this {@link ConnectionPlugin} does not support the
    *                                       requested strategy
    */
-  HostSpec getHostSpecByStrategy(final HostRole role, final String strategy)
+  HostSpec getHostSpecByStrategy(final HostRole role, final String strategy, final Properties props)
```

Comment:
I couldn't find a reason why you're extending this method with properties. Can double check?

Reply:
AH! Before the `HostSpecBuilder`, I was passing properties everywhere. After implementing `HostSpecBuilder` and replacing the `HostSpec` constructors everywhere, I think i forget to undo a bunch of the methods i modified to pass properties. :(

Reply:
Fixed! Removed the extra method param.